src/js/subtraction/components/SubtractionList.tsx

```diff
@@ -0,0 +1,51 @@
+import { map } from "lodash";
+import React, { useEffect } from "react";
+import { Badge, LoadingPlaceholder, NoneFoundBox, ViewHeader, ViewHeaderTitle } from "../../base";
+import { useUrlSearchParams } from "../../utils/hooks";
+import { useFetchSubtractions } from "../querys";
+import { SubtractionMinimal } from "../types";
+import { SubtractionItem } from "./SubtractionItem";
+import SubtractionToolbar from "./SubtractionToolbar";
+
+function renderRow({ created_at, user, name, id, ready, job }: SubtractionMinimal) {
+    return <SubtractionItem key={id} id={id} user={user} name={name} created_at={created_at} ready={ready} job={job} />;
+}
+
+/**
+ * A list of subtractions
+ */
+export default function SubtractionList() {
+    const [value, setValue] = useUrlSearchParams("find");
```

Comment:
```suggestion
    const [term, setTerm] = useUrlSearchParams("find");
```
Not descriptive enough